packages/suite-desktop-core/webpack/core.webpack.config.ts

```diff
@@ -112,6 +113,8 @@ const config: webpack.Configuration = {
         alias: {
             '@emurgo/cardano-serialization-lib-nodejs': '@emurgo/cardano-serialization-lib-browser',
             '@trezor/connect$': '@trezor/connect/src/index', // alternative for "module": "src/index" in connect's package.json
+            // optional dependency of systeminformation lib mocked as undefined (webpack needs it when targetting ES)
+            'osx-temperature-sensor': false,
```

Comment:
unlike other deps in `externals`, this one needs to be mocked as undefined, otherwise webpack crashes during building the desktop core `@trezor/suite-desktop-core build:app:dev`